### DISCUSSION TITLE
Inventory Item Description

### DISCUSSION THREAD

**Discussion by Ryebread4** [ORelio/Minecraft-Console-Client#1905] [Q&A]
I'm trying to make a bot that opens auction house and checks if anything posted meets a specified criteria. When you hover over the item in game it will show a "price" and I want to make it so it checks the items description (im assuming that's where its stored. Either that or the NBT data) and looks for "price" and check if its under a specific price. Any help is very much appreciated.

The little that I have so far.
https://pastebin.com/HqLL1g6Q

![image](https://user-images.githubusercontent.com/7307332/148969725-abf757f6-4871-4909-926e-fa04b9e1fa23.png)

**Comment by Daenges** [comment]
You need to write a C# script for this.
Every item in a Container has a [Lores](https://github.com/MCCTeam/Minecraft-Console-Client/blob/master/MinecraftClient/Inventory/Item.cs#L77) attribute, which should return the whole text in a list of strings.

**Reply by Ryebread4** [reply]
Yea im writing it in C# I just can't get the grasp of getting the lore of an inventory slot and posting it to console. Was just wondering if anyone already knew how to get item lores/info out of the inventory

**Reply by Daenges** [reply]
You can take a look into the [/inventory](https://github.com/MCCTeam/Minecraft-Console-Client/blob/master/MinecraftClient/Commands/Inventory.cs) command. This provides some examples on how to handle container interactions. And as I stated before, the item class has an attribute which returns the lore.

**Reply by Ryebread4** [reply]
I never even thought about lookin at the actual /inventory command, thanks. Also I saw the lores attribute thats where I got the idea in the description. My question was more of how to use it in code like how would I end up calling that function from my script. Im looking at the command now hopefully it helps me.
